core-bundle/src/Migration/MigrationInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Migration;
+
+interface MigrationInterface
+{
+    public function getName(): string;
```

Comment:
would it be a problem if multiple migrations have the same name?

Reply:
Yes, as the names are listed when the command asks you if you want to execute the migrations.

Reply:
well that's just unfortunate. But technically it won't be a problem? Like we're not storing the name of whether something was already executed?

Reply:
Correct. Technically duplicate names would not cause issues.